internal/x/io/prompter.go

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2017-present SIGHUP s.r.l All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package iox
+
+import (
+	"bufio"
+	"strings"
+)
+
+type Prompter struct {
+	Reader *bufio.Reader
+}
+
+func NewPrompter(r *bufio.Reader) *Prompter {
+	return &Prompter{
+		Reader: r,
+	}
+}
+
+func (p *Prompter) Ask(w string) bool {
+	response, err := p.Reader.ReadString('\n')
+	if err != nil {
+		return false
```

Comment:
it would be better not to hide the errors: if it fails here we could have a fairly hard time understanding where it broke.